README.md

```diff
@@ -280,8 +280,8 @@ request.send(completionHandler: {(response:Response?, error:Error?) in
 ## License
 This package contains code licensed under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 and may also view the License in the LICENSE file within this package.
 
-[img-bluemix-powered]: https://img.shields.io/badge/bluemix-powered-blue.svg
-[url-bluemix]: http://bluemix.net
+[img-ibmcloud-powered]: https://img.shields.io/badge/ibmcloud-powered-blue.svg
```

Comment:
I think I prefer the capitalized, spaced version `IBM%20Cloud`. Are there similar markdown files that display it your way?

Reply:
To keep it consistent with Node SDK, I am displaying `ibmcloud powered`

Reply:
<img width="637" alt="screen shot 2018-06-05 at 11 13 50 am" src="https://user-images.githubusercontent.com/8973287/40988774-8eaa9b7c-68b1-11e8-83b3-0a6c958f08b4.png">

Where do you mean? This is master/development as I see it

Reply:
We are changing the NPM module name to `ibmcloud-appid`. To reflect the change, I am updating the icon to `ibmcloud powered`. Also starter repo has a single icon for all the sample apps
<img width="1053" alt="screen shot 2018-06-05 at 11 24 58 am" src="https://user-images.githubusercontent.com/26394621/40989355-2a6a4fd4-68b3-11e8-80fb-108208189212.png">

To maintain consistency, I kept it as `ibmcloud`. If required we can update it.

Reply:
I don't really see the need for those to match. The readmes should reflect our corporate branding, which is `IBM Cloud`. In the past, we have join words others words like `appid` in repo/field names likely to limit the number of dashes, but whenever there is an actually branding situation the proper format is displayed (as in the line above the powered by bluemix in your screenshot).